packages/discovery-provider/plugins/notifications/src/email/notifications/renderEmail.ts

```diff
@@ -87,38 +85,30 @@ const DEFAULT_TRACK_COVER_ART_URL = ''
 const DEFAULT_PLAYLIST_IMAGE_IRL = ''
 
 const getUserProfileUrl = (user: UserResource) => {
-  let profilePictureUrl = DEFAULT_PROFILE_IMG
-  const contentNode = getContentNode()
   if (user.profile_picture_sizes) {
-    profilePictureUrl = `${contentNode}/content/${user.profile_picture_sizes}/1000x1000.jpg`
+    return formatImageUrl(user.profile_picture_sizes, 1000)
```

Comment:
good fixes!

i wonder if we could depend on API for this down the line

Reply:
yeah agreed that seems like the move long term